src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
+	private ArrayList<Integer> numberOfChars;
+	private ArrayList<String> IIN;
```

Comment:
same comment as above, please don't use IIN as a shortcut here, I'd rename this to `issuerIdentificationNumbers`